### DISCUSSION TITLE
Combine two pull requests to one

### DISCUSSION THREAD

**Discussion by v-ashwinik** [community/community#22827] [New to GitHub]
<p>Hi,<br>
I am new to GitHub, I have two pull requests from the same branch and the same person. Is it possible to combine the two pull requests to one? The changes are on the same content as well.<br>
Appreciate your inputs.</p>
<p>Thank you.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Technically: Just ask the person to combine the changes and push to one of the branches the pull requests are based on, or <a href="https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally">check out the pull requests locally</a> and do it yourself (that may require a new pull request).</p>
<p>That said, it’s often a good idea to keep changes small and merge them step by step. I don’t know your situation and code, so I can’t say which makes sense here.</p>
